src/routes/results.js

```diff
@@ -1,32 +1,34 @@
 import React from 'react';
-import { Button, StyleSheet, Text, View, Image } from 'react-native';
+import { Button, StyleSheet, View, Image } from 'react-native';
 
 import colors from '../assets/colors.js'
 import images from '../assets/images.js'
-import metrics from '../themes/metrics.js'
 
 import Navbar from '../components/Navbar.js'
 import FullWidthContainer from '../components/FullWidthContainer.js'
+import {standardStyle} from '../assets/styles.js'
 import LargeText from '../components/LargeText.js'
 
 class Results extends React.Component {
   render() {
     return ( //The info here is hardcoded, but we will enevtually want to make the info change depending on the item scanned.
-      <View
-        style={styles.results}>
-        <Text style={styles.headerUpper}>How to recycle</Text>
-        <Text style={styles.headerLower}>Beverage cans</Text>
-        <FullWidthContainer flex={containerFlex}>
-          <View style={{flexDirection: 'column'}}>
-            <Image source={images.recycleSymbol}/>
-            <Text>In recycling</Text>
-          </View>
-          <Text>Empty soda and bevrage cans should be rinsed first, then put in recycling</Text>
-        </FullWidthContainer>
-        <Button
-          title='Go To Landing'
-          onPress={() => this.props.navigation.navigate('Landing')}
-        />
+      <View style = {standardStyle.wrapper}>
+        <View style={standardStyle.container}>
+          <LargeText style={styles.headerUpper} text = "How to recycle"/>
+          <LargeText style={styles.headerLower} text = "Beverage cans"/>
+          <FullWidthContainer flex={containerFlex}>
+            <View style={{flexDirection: 'row', alignItems: 'center'}}>
+              <Image source={images.recycleSymbol} />
+              <LargeText text = "In recycling" style={{flex: 2}}/>
```

Comment:
I know this is just the quick code I sent you, but let's pull out these inline styles into a style sheet and try something like this:
```
          <View style={styles.result}>
              <Image source={images.recycleSymbol} style={styles.icon}/>
              <LargeText text = "In recycling" style={styles.instructionHeader}/>
            </View>
```
```
instruction: {flexDirection: 'row', alignItems: 'center'},
instructionHeader: {flex: 2},
icon: {width: 45} // I think this is their size, double check
```